### PR TITLE
GITOPS-1285 set equal width for details page card

### PR DESCRIPTION
**Pull request by ciiay**
This PR is for Epic [GITOPS-1276](https://issues.redhat.com/browse/GITOPS-1276), to add sync status to Resources on Environment Details page.

Note:
#10420 should be merged first and I will pull in all refactor changes to keep this PR up to date.

Jira Ticket:
GITOPS-[1285](https://issues.redhat.com/browse/GITOPS-1285)

This PR is to set equal width for cards on GitOps details page.

### CODE REVIEW

**Comment by keithchong** [comment]
Hi @divyansh42, @vikram-raj , while the tests are being looked at, could you review this?  We'd like to get this completed for this sprint.

**Review by vikram-raj — APPROVED** [review]
Verified. It looks as expected.
/lgtm

Screenshot 2021-11-26 at 8 05 36 PM

**Comment by reginapizza** [comment]
Since this doesn't have a bugzilla attached to it to get the bugzilla/valid-bug label, @Preeticp for this PR we need the `/docs-approved` label and @RickJWagner we need the `px-approved` label (even though it's just a bug fix, the CI bot says we need the labels), and we still need a `qe-approved` label on this as well, but I'm not sure who would provide that.

**Comment by keithchong** [comment]
Hi @vikram-raj , looks like we still need the labels:

approved docs-approved

I don't think docs need to be changed for this bug fix.

File: frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.tsx [hunk 1]
```diff
@@ -113,7 +113,8 @@ const GitOpsResourcesSection: React.FC<GitOpsResourcesSectionProps> = ({
             </SplitItem>
           </Tooltip>
         )}
-        {degradedResources?.length === 0 && nonSyncedResources.length === 0 && <>&nbsp;</>}
+        {(degradedResources === null || degradedResources.length === 0) &&
```
Comment (andrewballantyne) [inline frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.tsx:116]: Probably could have just done:
```suggestion
        {(degradedResources?.length === 0) &&
```
But if you're sure you are always `null` and not `undefined` -- it won't matter

Comment (andrewballantyne) [reply frontend/packages/gitops-plugin/src/components/details/GitOpsResourcesSection.tsx:116]: Ah yeah, the code is either explicitly `null`, an array of data or an empty `[]` 👍

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**